test/system/continuous/run-bulk-ingest.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
```

Comment:
Could name this script `run-bulk-generate.sh` since it does not import/ingest data and only generates data to import.